pyrobosim/pyrobosim/navigation/execution.py

```diff
@@ -205,6 +251,65 @@ def validate_remaining_path(self) -> None:
 
             time.sleep(max(0, self.validation_dt - (time.time() - start_time)))
 
+    def detect_closed_hallway(self) -> None:
+        """
+        Get lidar measurement and determine if it's scanning a hallway.
+        If yes, it would update the robot recorded_closed_hallways knowledge.
```

Comment:
this is now "known" instead of recorded, right? I would say just describe it in words instead of the variable name in case it changes again.